content/docs/releases/v1.6.1.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Release v1.6.1"
+short = "v1.6.1"
+weight = 1
+date = "2022-10-17"
++++
+
+
+# NSM v1.6.1
+
+NSM v1.6.1 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.6.1).
+
+## Kubernetes Compatibility
+NSM v1.6.0 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/3271188621) with Kubernetes versions:
+
+- v1.25.2
+- v1.24.6
+- v1.23.12
+- v1.22.13
+- v1.21.14
+- v1.20.15
+
+And also with [Calico CNI plugin](https://www.tigera.io/project-calico/).
+
+
+
+## Public Cloud Capability
+
+NSM v1.6.0 has been tested on 
+- [GKE](https://github.com/networkservicemesh/integration-k8s-gke/actions/runs/3271189442)
+- [AKS](https://github.com/networkservicemesh/integration-k8s-aks/actions/runs/3271188767)
+- [AWS](https://github.com/networkservicemesh/integration-k8s-aws/actions/runs/3271190013)
+- [bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/actions/runs/3271190113)
+
+## Changes since v1.6.1
+
+### k8s versions update
+
+Dropped testing on k8s v1.18, v1.19.
```

Comment:
Could we way why?  If I recall it was to simplify kind examples that may be using metallb so they don't need to use openssl, correct?

Reply:
Yes

Reply:
Added a comment about it. Could you check?